presto-docs/src/main/sphinx/release/release-0.248.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.248
+=============
+
+**Highlights**
+==============
+* New aggregation function MAP_UNION_SUM.
+* New scalar function DESTRUCTURE_TDIGEST.
+* Support for session property managers, that can override session properties set by a user.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve output buffer memory tracking to reduce lock contention.
+* Add support for overriding session properties using session property managers :doc:`/admin/session-property-managers`. Setting `overrideSessionProperties` to true will cause the property to be overridden and remain overridden even if subsequent rules match the property but don't have `overrideSessionProperties` set.
```

Comment:
overrideSessionProperties should be in double angle quotes.

Reply:
done